steps/terraform.yaml

```diff
@@ -128,7 +136,7 @@ steps:
         -var env=${{ parameters.environment }}
         -var builtFrom=$(Build.Repository.Name)
         -var product=${{ parameters.product }} ${{ parameters.planCommandOptions }}
-        -var-file $(System.DefaultWorkingDirectory)/$(buildRepoSuffix)/environments/${{ parameters.environment }}/$(tfVarsName).tfvars
```

Comment:
this makes `tfVarsName` unused but doesn't clean up `scripts/build-resource-values.sh`

Reply:
updating...